code/go/0chain.net/smartcontract/storagesc/challenge.go

```diff
@@ -26,7 +26,7 @@ import (
 	"go.uber.org/zap"
 )
 
-func (sc *StorageSmartContract) completeChallengeForBlobber(
+func (ssc *StorageSmartContract) completeChallengeForBlobber(
```

Comment:
Please avoid unrelated refactoring. Not only can it lead to hard to find errors and obscure the real changes being made, but it can also make merging the PR a nightmare.

I cannot see any real changes in challenge.go, if so can you revert these changes.